script/nsis/installer.nsi

```diff
@@ -0,0 +1,53 @@
+Name    "Git LFS"
+OutFile "GitLFSInstaller.exe"
+
+
+InstallDir $DESKTOP
+
+VIProductVersion "0.6.0.0"
+VIAddVersionKey ProductName      "Git LFS"
+VIAddVersionKey Comments         "Git LFS tool"
+VIAddVersionKey CompanyName      "GitHub, Inc"
+VIAddVersionKey LegalCopyright   "GitHub, Inc"
+VIAddVersionKey FileDescription  "Git LFS Tool"
+VIAddVersionKey FileVersion      1
+VIAddVersionKey ProductVersion   1
+VIAddVersionKey InternalName     "Git LFS"
+VIAddVersionKey LegalTrademarks  "something, something, darkside"
+VIAddVersionKey OriginalFilename "GitLFSInstaller.exe"
+
+Function .onGUIInit
+  IfFileExists $PROGRAMFILES64\Git\mingw64\bin\git.exe 0 +3
```

Comment:
I believe that's a wrapper which just starts up Git Bash and runs the real git.exe from the bin dir, so it'll probably be fine to just pull out ..\bin from directories which end in cmd.